app/views/admin/attachments/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<% if @attachment.errors.present? %>
+  <% @attachment.errors.full_messages.each do |message| %>
+    <%= message %>
+  <% end %>
+<% end %>
+
+<p class="title">Add an attachment:</p>
+
+<div class="block-form">
+  <%= form_for @attachment, url: { controller: "admin/attachments", action: "create" } do |f| %>
+    <%= f.label :name, 'Name:' %>
+    <%= f.text_field :name %>
+    <br />
+    <%= f.label :media_file, 'Attachment:' %>
```

Comment:
you have identical code in new/edit forms, move same code to form partial

Reply:
fixed